docs/building-images.md

```diff
@@ -157,7 +157,46 @@ operatingSystem:
   * `additionalRepos` - Optional; List of third-party RPM repositories that will be added to the package manager of the OS.
   * `sccRegistrationCode` - Optional; SUSE Customer Center registration code, used to connect to SUSE's internal RPM repositories.
 
-## SUSE Manager (SUMA)
+### Kubernetes
+
+The Kubernetes configuration section is another entirely optional one.
+It contains all necessary settings to configure and bootstrap a Kubernetes cluster.
+
+```yaml
+kubernetes:
+  version: v1.29.0+rke2r1
+  network:
+    apiVIP: 192.168.122.100
+    apiHost: api.cluster01.hosted.on.edge.suse.com
+  nodes:
+    - hostname: node1.suse.com
+      type: server
+    - hostname: node2.suse.com
+      type: server
+      initializer: true
+    - hostname: node3.suse.com
+      type: agent
+    - hostname: node4.suse.com
+      type: server
+    - hostname: node5.suse.com
+      type: agent
+  manifests:
+    urls:
+      - https://k8s.io/examples/application/nginx-app.yaml
+```
```

Comment:
I'd say to move the example after the explanation and to add an example of single node as well just in case.

Reply:
Examples will be added with https://github.com/suse-edge/edge-image-builder/issues/207. They don't belong in the schema definition / documentation.